api-specs/Gateway-EE/3.5/kong-ee-3.5.yaml

```diff
@@ -10156,88 +11210,2472 @@ paths:
+      operationId: get-admins-name_or_id-workspaces
+      description: Return workspaces associated with an admin.
+  '/debug/cluster/control-planes-nodes/log-level/{log_level}':
+    parameters:
+      - schema:
+          type: string
+          enum:
+            - debug
+            - info
+            - notice
+            - warn
+            - error
+            - crit
+        name: log_level
+        in: path
+        required: true
+        description: The log level
+    put:
+      summary: Set Node Log Level of All Control Plane Nodes
+      operationId: put-debug-cluster-control-planes-nodes-log-level-log_level
+      responses:
+        '200':
+          description: log level changed
+      description: |-
+        Change the log level of all Control Plane nodes deployed in Hybrid (CP/DP) cluster.
+        Care must be taken when changing the log level of a node to debug in a production environment because the disk could fill up quickly. As soon as the debug logging finishes, revert back to a higher level such as notice.
+        It’s currently not possible to change the log level of DP and DB-less nodes.
+        This endpoint can be protected with RBAC, and changes will be reflected in the audit logs.The log level change is propagated to all Nginx workers of a node, including to newly spawned workers.
+
+        Log levels are set in Kong’s configuration. Following are the log levels in increasing order of their severity: `debug`, `info`, `notice`, `warn`, `error` and `crit`. For more information review the [logging reference](https://docs.konghq.com/gateway/latest/production/logging/).
+
+        Currently, when a user dynamically changes the log level for the entire cluster, if a new node joins the cluster, the new node will run at the previous log level, not at the log level that was previously set dynamically for the entire cluster. To work around that, make sure the new node starts with the proper level by setting the startup kong.conf setting [KONG_LOG_LEVEL](https://docs.konghq.com/gateway/latest/reference/configuration/#log_level).
+      tags:
+        - debug
+  '/debug/cluster/log-level/{log_level}':
+    parameters:
+      - schema:
+          type: string
+          enum:
+            - debug
+            - info
+            - notice
+            - warn
+            - error
+            - crit
+        name: log_level
+        in: path
+        required: true
+        description: The log level
+    put:
+      summary: Set Node Log Level of All Nodes
+      operationId: put-debug-cluster-log-level-log_level
+      responses:
+        '200':
+          description: log level changed
+      tags:
+        - debug
+      description: |-
+        Change the log level of all nodes in a cluster.
+        Care must be taken when changing the log level of a node to debug in a production environment because the disk could fill up quickly. As soon as the debug logging finishes, revert back to a higher level such as notice.
```

Comment:
```suggestion
        Be careful when changing the log level of a node to debug in a production environment because the disk could fill up quickly. As soon as the debug logging finishes, revert back to a higher level, such as notice.
```